vespalib/src/vespa/fastlib/text/unicodeutil.cpp

```diff
@@ -6,28 +6,6 @@
 #include "unicodeutil-charprops.cpp"
 #include "unicodeutil-lowercase.cpp"
 
-namespace {
-
-class Initialize
-{
-public:
-    Initialize() noexcept { Fast_UnicodeUtil::InitTables(); }
-};
-
-Initialize _g_initializer;
```

Comment:
This instance caused the constructor for `Initialize` to be called, thus `Fast_UnicodeUtil::InitTables()` is called once, before main is called at program startup time. But InitTables doesn't seem to do anything useful.